features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/store-new/jaggery.conf

```diff
@@ -0,0 +1,60 @@
+{
+    "errorPages":
+         {
+            "401":"/site/pages/error-pages/401.html",
+            "403":"/site/pages/error-pages/403.html",
+            "404":"/site/pages/error-pages/404.html",
+            "500":"/site/pages/error-pages/500.html",
+            "0":"/site/pages/error-pages/error-page.html"
+         }
+    ,
+
```

Comment:
We can remove these unnecessary new lines.

Reply:
Has been removed with commit [05721a7449c0fdd5ea8bf15b61ce3da047f9bba2](https://github.com/wso2/carbon-apimgt/commit/05721a7449c0fdd5ea8bf15b61ce3da047f9bba2)